tests/cli/test_cli.py

```diff
@@ -1329,6 +1329,7 @@ def test_cli_connections_list_redirect(self):
             self.assertEqual(0, proc.returncode)
 
     def test_cli_connections_add_delete(self):
+        db.resetdb()
```

Comment:
Can you add a `# TODO: ...` here so we stand a chance of remembering this if we don't get straight on to the other fix.

The other option would be to mock the Connection adding code -- we don't need to test that here, just the CLI code right?

Reply:
In most cases, CLI only has integration tests. We do not test CLI with unit tests and very rarely use mock. I think this is a good approach because we don't have a lot of these tests, but they provide a lot more confidence. If we develop CLI then we will have to think that the next tests will be only unit tests.

Reply:
I added TODO comment.